content/chef_compliance_phase.md

```diff
@@ -420,6 +424,29 @@ Setting the attribute `default['audit']['insecure']` to `true` will skip SSL cer
 default['audit']['insecure'] = true
 ```
 
+### interval
+
+**New in Chef Infra Client 17.8.29**
+
+The `node['audit']['interval']` attribute allows you to control the frequency of Compliance Phase scans independently of Chef Infra Client runs. This helps control the impact of compliance scans impact on system performance in business environments that require compliance scans less frequently than Chef Infra Client Runs.
+
+`node['audit']['interval']['enabled']`
+: Set to true to enable interval runs.
+
+  ```ruby
+  # Enable independent Compliance Phase scans
+  node['audit']['interval']['enabled'] = true
```

Comment:
Matching what we're doing everywhere else. Looks like most of our examples assume attributes files.

```suggestion
  default['audit']['interval']['enabled'] = true
```